schema/deploy/trigger_functions/commit_form_change.sql

```diff
@@ -38,16 +35,28 @@ begin
   vals := (select array_to_string(array(select quote_nullable(value) from jsonb_each_text(new.new_form_data)), ','));
 
   if new.operation = 'INSERT' then
```

Comment:
Do we need to add support for lowercase? It's the team's coding style, and I see we have it in a few tests below too

Reply:
the `operation` column should really be an enum or foreign key to a table, not a varchar, but that's a different PR